Index: python3-thingsboard-gateway-2.6/thingsboard_gateway/tb_utility/tb_utility.py
===================================================================
--- python3-thingsboard-gateway-2.6.orig/thingsboard_gateway/tb_utility/tb_utility.py
+++ python3-thingsboard-gateway-2.6/thingsboard_gateway/tb_utility/tb_utility.py
@@ -90,7 +90,7 @@ class TBUtility:
                         return TBUtility.loaded_extensions[extension_type + module_name]
                     if path.exists(extension_path):
                         for file in listdir(extension_path):
-                            if not file.startswith('__') and file.endswith('.py'):
+                            if not file.startswith('__') and (file.endswith('.py') or file.endswith('.pyc')):
                                 try:
                                     module_spec = util.spec_from_file_location(module_name, extension_path + path.sep + file)
                                     log.debug(module_spec)
